src/Motives/Motives.js

```diff
@@ -1,44 +1,125 @@
-import React, { useEffect, useState } from 'react'
-import { useHistory } from 'react-router-dom'
-import MotiveButtons from './MotiveButtons'
-import { evaluateMatchingStyles } from './util'
+import React from 'react'
+import { DragDropContext, Droppable } from 'react-beautiful-dnd'
+import styled from 'styled-components'
+import Button from '../common/Button'
+import FloatingButtonContainer from '../common/FloatingButtonContainer'
 import HeadlineUnderline from '../common/HeadlineUnderline'
+import { motiveData } from '../data/motiveData'
+import MotiveItem from './MotiveItem'
+import { MotivesList } from './MotivesList'
+import { useMotiveSelection } from './useMotiveSelection'
+import UserMotiveDroppableWrapper from './UserMotiveDroppableWrapper'
 
 export default function Motives() {
-  const [userMotives, setUserMotives] = useState([])
-  const history = useHistory()
-
-  useEffect(() => {
-    if (userMotives.length === 3) {
-      const userPersonalityStyles = evaluateMatchingStyles(userMotives)
-      const questionnaireUrl =
-        '/questionnaire/' + userPersonalityStyles[0] + userPersonalityStyles[1]
-      setUserMotives([])
-      history.push(questionnaireUrl)
-    }
-  }, [userMotives, history])
+  const {
+    onDragStart,
+    onDragEnd,
+    motives,
+    questionnaireUrl,
+    handleMotiveClick,
+  } = useMotiveSelection()
 
   return (
-    <>
-      <img alt="" src="/img/motives-intro.svg" />
+    <DragDropContext onDragStart={onDragStart} onDragEnd={onDragEnd}>
       <HeadlineUnderline>
         <h1>Schritt 1: Bedürfnisse</h1>
       </HeadlineUnderline>
-      <h2>
-        Bitte wählen Sie die 3 Bedürfnisse aus, die Ihnen am wichtigsten sind.
-      </h2>
-      <ol>
-        <li> {userMotives[0]} </li>
-        <li> {userMotives[1]} </li>
-        <li> {userMotives[2]} </li>
-      </ol>
-      <MotiveButtons handleMotiveClick={handleMotiveClick} />
-    </>
+      <SmallH2 as="h2">
+        Wählen Sie bitte intuitiv die drei Bedürfnisse aus, die für Sie am
+        Wichtigsten sind. Sie können die Bedürfnisse hin- und herziehen oder
+        anklicken.
+      </SmallH2>
+      <MotiveBG>
+        <UserMotiveDroppableWrapper
+          droppableId="slot1"
+          motives={motives}
+          handleMotiveClick={handleMotiveClick}
+        />
+        <UserMotiveDroppableWrapper
+          droppableId="slot2"
+          motives={motives}
+          handleMotiveClick={handleMotiveClick}
+        />
+        <UserMotiveDroppableWrapper
```

Comment:
It's the same code 3 times except for the id, maybe think about refactoring this.